<launch>
    <node pkg="fastlio" type="temp_node" name="temp_node" output="screen"/>
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fastlio)/rviz/localize.rviz" />
</launch>